tests/integration/urlconf_overrides/xss.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from django.conf.urls import url
+from django.http import HttpResponse
+
+
+# pylint: disable=unused-argument
+def view(*args, **kwargs):
+    return HttpResponse(status=200, content="<b>XSS Test</b>")
```

Comment:
@kaizoku I'd suggest `<script>alert("XSS")</script>` here - to make the point clear.